server/services/interventionsService.test.ts

```diff
@@ -698,4 +698,40 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
       expect(serviceCategory.desiredOutcomes).toEqual(desiredOutcomes)
     })
   })
+
+  describe('createReferral', () => {
+    beforeEach(async () => {
+      await provider.addInteraction({
+        state: 'a draft referral with ID dfb64747-f658-40e0-a827-87b4b0bdcfed exists and is ready to be submitted',
+        uponReceiving:
+          'a POST request to create a referral from the draft referral with ID dfb64747-f658-40e0-a827-87b4b0bdcfed',
+        withRequest: {
+          method: 'POST',
+          path: '/referral',
+          headers: { Accept: 'application/json', Authorization: `Bearer ${token}` },
+          body: { draftReferralId: 'dfb64747-f658-40e0-a827-87b4b0bdcfed' },
+        },
+        willRespondWith: {
+          status: 201,
+          body: Matchers.like({
+            id: '81d754aa-d868-4347-9c0f-50690773014e',
+            created: '2021-01-11T10:32:12.382884Z',
+            referenceNumber: 'HDJ2123F',
```

Comment:
Perhaps I should add examples of all the draft referral fields here to make it clear that the data is copied from the draft referral to the referral?

Reply:
yeh works for me

Reply:
or we can just not return anything in the response, and use the location header. and validate the response in a different test.

Reply:
It's a little easier for us to use the actual entity in the response, because after sending the referral I'm going to want to redirect to a different page like `/referrals/${sentReferralId}/confirmation` - this way I can just grab the `sentReferralId` from the response body instead of having to fetch the sent referral to find out its ID.

Reply:
Ah sorry, missed what you were talking about here - thought you were talking about the behaviour of the API, you were just talking about the test.

Reply:
I've updated the tests to contain a full example of a sent referral.